src/core_plugins/kibana/server/routes/api/suggestions/register_value_suggestions.js

```diff
@@ -10,7 +10,11 @@ export function registerValueSuggestions(server) {
 
       const { callWithRequest } = server.plugins.elasticsearch.getCluster('data');
       const include = query ? `.*${query}.*` : undefined;
-      const body = getBody({ field, include });
+      const body = getBody({
+        field,
+        include,
```

Comment:
I wonder if this parameter is partly what's hurting us. I don't know how it's implemented, but a regex with a leading wildcard is suspicious. Something to investigate when we try to fix the performance later.